.github/workflows/azure-static-web-apps-gentle-beach-0eb2d061e.yml

```diff
@@ -0,0 +1,50 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - Assignment1

```

Comment:
Should this be changed to another branch? Your Azure link doesn't seem to be matching your code.

Reply:
Yep, that's our mistake!
The Azure site should actually reflect our changes now lol

Reply:
Completly missed that lol